src/core/layout/scrollable-list/scrollable-list-helpers.js

```diff
@@ -4,12 +4,21 @@
  * @author BBC Children's D+E
  * @license Apache-2.0 Apache-2.0
  */
-
 /* eslint-disable no-console */
-export const onClick = gelButton => {
-    console.log(`Clicked ${gelButton.config.id}`); // placeholder
-};
 
-export const assetKey = (key, assetKeys) => {
-    return [assetKeys.prefix, key].join(".");
+/* gelButton's clickable zone extends beyond rexUI's mask [CGPROD-2795]*/
```

Comment:
Prob don't need this comment - if there's anything that's a bit of a workaround we usually make sure there's a described test for it.